fundamentals/apple/gettingstarted/final/Favourites/Shared/FavouriteNumberView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  FavouriteNumberView.swift
+//  Shared
+//
+//  Created by Peter Friese on 05.07.22.
+//  Copyright © 2021 Google LLC. All rights reserved.
```

Comment:
nit: I think `All rights reserved` is directly at odds with the license below, but I'm not a lawyer. Also copyright date should be 2022.